coralproject/talk
# Coral plugin Auth

## What does this PR do?
 - Abstracts Auth flow as a plugin `coral-plugin-auth`
 - Adds `embed`, `stream`, `login` slots

## Testing this PR
- Check that every auth path works as expected:
  - Login Flow
  - Sign Up Flow
  - Sign In Flow
  - Forgot Password
  - Change Username, etc.
